zulipterminal/ui_tools/buttons.py

```diff
@@ -221,6 +238,10 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             )
         elif is_command_key("STREAM_DESC", key):
             self.model.controller.show_stream_info(self.stream_id)
+        elif is_command_key("ENTER", key):
+            self.set_status(is_active=True)
+            if self.model.active_button:
+                self.model.active_button.set_status(is_active=False)
```

Comment:
This is still duplicate functionality, to that taking place in the model which should cover all stream narrowing behaviors.